### PR TITLE
Configure docker compose

### PR DESCRIPTION
**Pull request by gregoryguillou**

### CODE REVIEW

File: cli/cmd/configure.go [hunk 1]
```diff
@@ -164,6 +164,11 @@ func connectAPI(cfg config) (string, error) {
 		return "", err
 	}
 
+	if resp.StatusCode != 200 {
+			fmt.Println("Unauthorized Access, check your credentials")
```
Comment (jney) [inline cli/cmd/configure.go]: use `go fmt` for  automatic formatting. the indentation in weird

File: cli/cmd/net.go [hunk 2]
```diff
@@ -44,15 +43,20 @@ func get(url string, display bool) (map[string]interface{}, error) {
...
+		}
+
+		if display {
+			s, _ := prettyjson.Marshal(dat)
+			fmt.Println(string(s))
+		}
+		return dat, nil
+	} 
+	fmt.Println(string(data))
+	return nil, nil
```
Comment (jney) [inline cli/cmd/net.go]: is it normal to return `nil, nil`?
by the way, in this function you never return an error, maybe you don't want to. so change the signature of the function

File: cli/cmd/net.go [hunk 3]
```diff
@@ -44,15 +43,20 @@ func get(url string, display bool) (map[string]interface{}, error) {
...
-
-	if display {
-		s, _ := prettyjson.Marshal(dat)
-		fmt.Println(string(s))
-	}
-	return dat, nil
+  if resp.StatusCode == 200 {
+		var dat map[string]interface{}
+		if err := json.Unmarshal(data, &dat); err != nil {
+			panic(err)
```
Comment (jney) [inline cli/cmd/net.go]: be carefull with panic, it exits your program

File: cli/cmd/configure.go (hunk 1 above)
Comment (gregoryguillou) [reply cli/cmd/configure.go]: fixed

File: cli/cmd/net.go (hunk 2 above)
Comment (gregoryguillou) [reply cli/cmd/net.go]: I've removed the err. You are right

File: cli/cmd/net.go (hunk 3 above)
Comment (gregoryguillou) [reply cli/cmd/net.go]: Yes, I want to crash when it fails (at least for now)

**Comment by gregoryguillou** [comment]
This PR closes #90

**Pull request merged**